src/main.js

```diff
@@ -1,11 +1,11 @@
-// Datafeed implementation, will be added later
+// Datafeed implementation
 import Datafeed from './datafeed.js';
 
 window.tvWidget = new TradingView.widget({
-	symbol: 'Bitfinex:BTC/USD', // default symbol
-	interval: '1D', // default interval
-	fullscreen: true, // displays the chart in the fullscreen mode
-	container: 'tv_chart_container',
+	symbol: 'Bitfinex:BTC/USD',             // Default symbol
+	interval: '1D',                         // Default interval
+	fullscreen: true,                       // Displays the chart in the fullscreen mode
+	container: 'tv_chart_container',        // Reference to the attribute of the DOM element
```

Comment:
It's not necessarily true; _an_ is more generic in the sense that you could have more than 1 container in your DOM and therefore decide which element attaching the chart to.
Having said that I don't think it matters that much. The general documentation is clear enough and we never have questions around that so probably not worth spending too much time rephrasing everything.